spec/controllers/pull_requests_controller_spec.rb

```diff
@@ -11,22 +11,20 @@
         end
       end
 
-      let(:repository_fields) { Hash.new }  # expected to be overwritten by a sub-context
       let!(:repository) do
-        FactoryGirl.create(:repository,
-                           { url: "git@git.example.com:square/web.git" }.merge(repository_fields)
-                          )
+        FactoryGirl.create(:repository, { url: "git@git.#{git_server_type}.com:square/web.git" }.merge(repository_fields))
       end
+      let(:repository_fields) { Hash.new }  # expected to be overwritten by a sub-context
 
-      context "for push events" do
+      context "when push requests come" do
         let(:repository_fields) { { run_ci: true } }
         let!(:master_branch) { FactoryGirl.create(:master_branch, repository: repository) }
         let!(:convergence_branch) { FactoryGirl.create(:branch, repository: repository, convergence: true, name: "convergence_branch") }
 
-        it "creates a build" do
+        it "should create a build for stash push request" do
```

Comment:
the description should not have "stash" in it as this will run for both github and stash.